docs/index.rst

```diff
@@ -2,42 +2,64 @@
 sunpy core Documentation
 ************************
 
-The sunpy core package is a community-developed, free and open-source solar
+``sunpy`` is a community-developed, free and open-source solar
 data analysis environment for Python.
 
+New Users
+*********
+
+If you are a first time user or new to Python,
+
+..
+        * :Introduction to sunpy core [new page described in #6615]
+
+* :doc:`installation guide <intro/installation>`
+
+Our **tutorial** provides a walkthrough on how to use the key features of sunpy core, we recommend new users start here.
+
+* :doc:`Getting Started <intro/index>`
```

Comment:
I'm not a fan of this manual ordering and then a hidden toc tree. How about we move "Installation guide" to a new page that shows up in the side bar, then include that as a toctree above, and turn this getting started into a toctree?

Reply:
See my comment on the main thread. I actually think there's potential value in the separation.

In this case I think not having installation guide in the sidebar is an improvement (I got rid of it in #6613)